lib/dhcp_wire.ml

```diff
@@ -696,7 +698,80 @@ let options_of_buf buf buf_len =
       | 74 ->  take (Irc_servers (get_ip_list ()))
       | 75 ->  take (Streettalk_servers (get_ip_list ()))
       | 76 ->  take (Streettalk_da (get_ip_list ()))
+      | 77 ->  take (User_class (get_string ()))
+      | 78 ->  take (Directory_agent (get_string ()))
+      | 79 ->  take (Service_scope (get_string ()))
+      | 80 ->  take Rapid_commit
+      | 81 ->  take (Client_fqdn (get_string ()))
+      | 82 ->  take (Relay_agent_information (get_string ()))
+      | 83 ->  take (Isns (get_string ()))
+      | 85 ->  take (Nds_servers (get_string ()))
+      | 86 ->  take (Nds_tree_name (get_string ()))
+      | 87 ->  take (Nds_context (get_string ()))
+      | 88 ->  take (Bcmcs_controller_domain_name_list (get_string ()))
+      | 89 ->  take (Bcmcs_controller_ipv4_addrs (get_ip_list ~min_len:1 ()))
```

Comment:
min_len in this function is not List.length, it is the minimum size of the object in bytes, so the correct should be the default min_len:4, which is one ip.